contingency/contingency-api/src/main/java/com/powsybl/contingency/contingency/list/LineCriterionContingencyList.java

```diff
@@ -6,42 +6,25 @@
  */
 package com.powsybl.contingency.contingency.list;
 
-import com.powsybl.contingency.contingency.list.criterion.PropertyCriterion;
-import com.powsybl.contingency.contingency.list.criterion.RegexCriterion;
-import com.powsybl.contingency.contingency.list.criterion.SingleNominalVoltageCriterion;
-import com.powsybl.contingency.contingency.list.criterion.TwoCountriesCriterion;
+import com.powsybl.contingency.contingency.list.criterion.*;
 import com.powsybl.iidm.network.IdentifiableType;
 
 import java.util.List;
 
 /**
  * @author Etienne Lesot <etienne.lesot@rte-france.com>
  */
-public class LineCriterionContingencyList extends AbstractEquipmentCriterionContingencyList {
-    private final TwoCountriesCriterion twoCountriesCriterion;
-    private final SingleNominalVoltageCriterion singleNominalVoltageCriterion;
+public class LineCriterionContingencyList extends AbstractLineCriterionContingencyList {
 
     public LineCriterionContingencyList(String name,
                                         TwoCountriesCriterion twoCountriesCriterion,
-                                        SingleNominalVoltageCriterion singleNominalVoltageCriterion,
+                                        TwoNominalVoltageCriterion twoNominalVoltageCriterion,
```

Comment:
Do we want to keep as deprecated the previous constructor?

Reply:
no